hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3AConfiguration.java

```diff
@@ -358,7 +359,7 @@ public void shouldBeAbleToSwitchOnS3PathStyleAccessViaConfigProperty()
     try {
       fs = S3ATestUtils.createTestFileSystem(conf);
       assertNotNull(fs);
-      S3Client s3 = fs.getAmazonS3ClientForTesting("configuration");
+      S3Client s3 = getS3Client("configuration");
```

Comment:
what about just force disabling CSE by removing base/bucket overrides and setting to false?